autopts/wid/mesh.py

```diff
@@ -958,6 +958,13 @@ def hdl_wid_255(_: WIDParams):
     """
     return True
 
+def hdl_wid_234(_: WIDParams):
+    """
+    Implements:
+    description: Please confirms that the IUT receives the beacon with iv index
+                 set to 12
+    """
+    return True
```

Comment:
It would be nice to have some verification here, but in Zephyr there is no callback for receiving beacons, IVI and other things from packet are just pulled and passed into `net_beacon_resolve`, and I don't see there any callback either, everything is done in backend. I guess we need to live with this like that. `bt_mesh_scan_cb` also doesn't give app access to itself, so we cannot see what Mesh scanned (maybe it's good, it'd be a lot of events)

Reply:
I created an issue to add API for testing this: https://github.com/zephyrproject-rtos/zephyr/issues/55366

Reply:
I considered the same concerns, however the latter steps in test implicitly verifies this step. But having an method to access that info from stuck better help to make sure intermediate steps.